packages/plugin-core/src/commands/PublishDevCommand.ts

```diff
@@ -9,13 +9,20 @@ import fs from "fs-extra";
 import _ from "lodash";
 import { window } from "vscode";
 import { DENDRON_EMOJIS } from "@dendronhq/common-all";
+import { IDendronExtension } from "../dendronExtensionInterface";
 
 type CommandOutput = {
   pid: number;
 };
 
 export class PublishDevCommand extends BasicCommand<CommandOutput> {
   key = DENDRON_COMMANDS.PUBLISH_DEV.key;
+  protected extension: IDendronExtension;
```

Comment:
Attracts caution to re-declare the same field that exists in the base class, any reason why we can't use the `extension` field that is coming from the base class?